README.md

```diff
@@ -1,2 +1,66 @@
-# webdriver_kaifuku
-restartable webdriver instances
+<h1 align="center"> webdriver_kaifuku </h1>
+<h3 align="center"> Restartable webdriver instances </h3>
```

Comment:
Why html?

Reply:
I can say copy from my other repo

but If I remember correctly; I used HTML for central alignment. 
I think markdown or rst no support such styling.

https://github.com/RonnyPfannschmidt/webdriver_kaifuku/blob/869963993b299afc2df003a4f8870010ab0ebf7d/README.md

@RonnyPfannschmidt If I remember correctly I was inspired by psf `black` project.

Reply:
@RonnyPfannschmidt can I change it to markdown?

Reply:
thanks for the ping, no its fine